client/src/components/App.js

```diff
@@ -1,5 +1,5 @@
 import React, { useState, useEffect } from "react"
-import { BrowserRouter as Router, Switch, Route } from "react-router-dom"
+import { BrowserRouter as Router, Switch, Route, useParams } from "react-router-dom"
```

Comment:
It looks like `useParams` is never being used in this file so we can remove it from the import statement.

Reply:
Awesome! So can we just remove that from the statement to resolve?

Reply:
deleted!